Makefile

```diff
@@ -114,14 +117,16 @@ test-system-default-registry:
 	@go test -timeout=45m -v -tags=systemdefaultregistry -count=1 ./entrypoint/airgap/... -destroy ${DESTROY}
 
 #========================= TestCode Static Quality Check =========================#
-pre-commit: go-check shell-check
+pre-commit: go-check
```

Comment:
Why are we removing shell-check from here?

Reply:
Adding in a later PR where I will also overhaul the scripts directory contents.